.circleci/config.yml

```diff
@@ -185,22 +191,35 @@ jobs:
               exit 1
             fi
           name: Tag Release in GitHub
+      - run:
+          command: |
+            echo {} > package.json
+            version=$(jq -r .version composer.json)
+            ~/.yarn/bin/yarn add github-release-notes
+            ~/.yarn/bin/yarn gren release -m -t $version -T $GITHUB_ACCESS_TOKEN
+            ~/.yarn/bin/yarn gren release -o -t $version -T $GITHUB_ACCESS_TOKEN
+          name: Create GitHub Release Via Gren
   publish:
     docker:
       - image: 244249143763.dkr.ecr.us-west-2.amazonaws.com/ns8-php
     steps:
       - checkout
       - run:
-          command: zip -x '.*' -x CODEOWNERS -x 'Test/*' -r9 NS8_Protect.zip .
+          command: zip -x 'yarn*' -x -x '.*' -x package.json -x '*node_modules*' -x CODEOWNERS -x 'Test/*' -r9 NS8_Protect.zip .
```

Comment:
This double `-x` doesn't look right. Also I think you won't need to add any exclusions here if you install `gren` globally.

Reply:
I'll try again with gren globally. I had originally and ran into issues but happy to try again. The package.json will need to be present though regardless for it to run in the dir